libshaderc/src/shaderc_test.cc

```diff
@@ -175,7 +180,8 @@ class CompileStringTest : public testing::Test {
                            options);
     EXPECT_TRUE(shaderc_module_get_success(comp.result())) << kind << '\n'
                                                            << shader;
-    return shaderc_module_get_bytes(comp.result());
+    return std::string(shaderc_module_get_bytes(comp.result()),
```

Comment:
This doesn't belong in the MR comments.  Put the comment in the code.
Also, see above, I argue that you should have two CompilationOutput methods: one for string, and another for the binary, so you can more naturally check the binary contents.